script/Arcueid/Day 5/05_00_ARC05_9_2B.txt

```diff
@@ -66,23 +73,25 @@ In the face of the towering wave, the man managed to regain his lapsed sense of
 [sha:839c9e5128a6b6b6535cd92c4b1928f700fe0392]{
 -- Page 664, Offset 10434.
 -- いや、正確には２分前から。
-// Reified verb from above いや、正確には２分前から。
+// Reified verb from above
 No, he regained it two minutes ago to be exact.
 }
 [sha:e47fc39eb406a0747647ec50cab6374fbf065c09]{
 -- Page 664, Offset 10435.
 -- 常に寒波に<侵|おか>され正気を無くした筈の脳が、あの一瞬―――目前に迫る人間から飛び退いた時から、正常に稼働しようと努めていた。
-His mind, which should have lost all sanity from the constant barrage of cold, has been trying to function normally ever since the moment he jumped away from the human closing in before him.
+His mind, which should have lost all sanity from the constant barrage of cold,
+ has been trying to function normally ever since that moment―――
+ The moment he jumped away from the human closing in on him.
```

Comment:
Continuation of line after sentence ended with em-dash is uncommon in the script, intentional? We usually have them either be terminal or no spaces either side (in which case probably no caps on `the moment`

Reply:
unintentional. artifact of a different structure i was toying with

Reply:
wait, no. reading back over, this dash series was specifying the moment. It was supposed to be 2x dashes and no caps.